### PR TITLE
Quantization with OpenVINO 2022.1

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Software Requirements 
-- TensorFlow 2.3.0
-- OpenVINO 2021.1
+- TensorFlow 2.5.3
+- OpenVINO 2022.1
+
+OpenVINO Development Tools [installation instruction](https://docs.openvino.ai/latest/openvino_docs_install_guides_install_dev_tools.html)
```
Comment (dkurt) [inline README.md:5]: Maybe we can install it just by pip?

File: README.md [hunk 2]
```diff
@@ -86,13 +78,13 @@ python3 evaluate.py --engine opvn --xml /model/optimized/resnet_v2_101_299_opt.x
...
+Сomparing the ```benchmark_app``` results we got an acceleration of about 1.9 times for **INT8 model**:
 
-|                  | Latency  | Throughput |
-|:----------------:|:--------:|:----------:|
-| OpenVINO model   | 45.98 ms | 21.75 FPS  |
-| INT8 model       | 29.22 ms | 34.23 FPS  |
+|                  | Latency (Median)  | Throughput |
+|:----------------:|:-----------------:|:----------:|
+| OpenVINO model   |     212.16 ms     | 18.02 FPS  |
+| INT8 model       |     112.41 ms     | 34.26 FPS  |
```
Comment (dkurt) [inline README.md:90]: Please add some info about CPU

File: README.md (hunk 1 above)
Comment (likholat) [reply README.md:5]: This instruction says how to install it with pip

**Comment by likholat** [comment]
@dkurt thanks for the review :)

**Pull request merged**